go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/GoogleCloudPlatform/opentelemetry-operations-collector
 
 go 1.18
 
+replace github.com/GoogleCloudPlatform/opentelemetry-operations-collector/receiver/nvmlreceiver => ./receiver/nvmlreceiver
+
 require (
+	github.com/GoogleCloudPlatform/opentelemetry-operations-collector/receiver/nvmlreceiver v0.61.0
```

Comment:
I'm not sure the proper way to do this since I don't see varnishreceiver here?

Reply:
This is only necessary for receivers outside of this repo. Receivers in the repo like `varnishreceiver` and this one are simply imported and don't need to be in the `go.mod`. It can be removed from here.

Reply:
Sorry, I didn't notice you also had a `replace` directive. If you didn't, that will need to be removed as well.

The reason it's not an issue for `varnishreceiver` is that it's simply imported in `components.go` as a package local to the current module, as should the `nvmlreceiver`.

Reply:
Okay, I removed it. 

Now I see:
![image](https://user-images.githubusercontent.com/4366369/196505372-db9a5fe2-0052-4582-a12c-8be652a85916.png)

Is that expected for now?

Reply:
Ah, I also didn't notice that you have a `go.mod` and `go.sum` in the `nvmlreceiver` folder. My mistake. Is the intention that we want to distribute this as a separate go module as a separate import from the overall module? I would assume the answer is no, but I wasn't part of any design discussions. 

If the answer is no, then the `go.mod` and `go.sum can also be removed, and its dependencies can be added into the main `go.mod`. If the answer is yes, then you can return the line we are commenting on and the `go replace`.

Reply:
Oh. No it should just be like varnishreceiver. Cool, that seems to have worked. I'm not an expert with the Go module system.